frontend/src/api/k8s/notebooks.ts

```diff
@@ -197,23 +197,23 @@ const getStopPatch = (): Patch => ({
   value: getStopPatchDataString(),
 });
 
-const getInjectOAuthPatch = (enableServiceMesh: boolean): Patch => ({
-  op: 'add',
-  path: '/metadata/annotations/notebooks.opendatahub.io~1inject-oauth',
-  value: String(!enableServiceMesh),
-});
-
-const getProxyInjectPatch = (enableServiceMesh: boolean): Patch => ({
-  op: 'add',
-  path: '/metadata/labels/sidecar.istio.io~1inject',
-  value: String(enableServiceMesh),
-});
-
-const getServiceMeshPatch = (enableServiceMesh: boolean): Patch => ({
-  op: 'add',
-  path: '/metadata/annotations/opendatahub.io~1service-mesh',
-  value: String(enableServiceMesh),
-});
+export const getServiceMeshPatches = (enableServiceMesh: boolean): Patch[] => [
```

Comment:
Simple inquiry -- do we plan to support these patches not being included or should they always be included since they share the on/off state.

Reply:
I think these patches should always be included. 

Reasoning being that in particular, the `inject-oauth` alongside the `service-mesh` patch/annotation is used by the `notebook-controller` (see PR https://github.com/opendatahub-io/kubeflow/pull/172/files) to decide if the oauth-proxy sidecar should be injected. 

So for inject-oauth particularly this must be included, and I think similar reasoning applies for the other two.